### PR TITLE
Validate e2e test env variables and update docs

### PR DESCRIPTION
**Pull request by marlosin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [our conventions](../CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[X] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: scripts/env-check.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const availableVars = [
+  "E2E_PROJECT_ID",
+  "NODE_TLS_REJECT_UNAUTHORIZED",
+  "JEXIA_DEV_DOMAIN",
+  "E2E_EMAIL",
+  "E2E_PASSWORD",
+  "RECAPTCHA_TOKEN",
+];
+
+const docUrl = 'https://github.com/jexia/jexia-sdk-js/blob/master/e2e/README.md'
```
Comment (KutsenkoA) [inline scripts/env-check.js]: double quotas :-D
and missed `;`

File: scripts/env-check.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const availableVars = [
+  "E2E_PROJECT_ID",
+  "NODE_TLS_REJECT_UNAUTHORIZED",
+  "JEXIA_DEV_DOMAIN",
+  "E2E_EMAIL",
+  "E2E_PASSWORD",
+  "RECAPTCHA_TOKEN",
```
Comment (KutsenkoA) [inline scripts/env-check.js]: for JFS tests we also need a bunch of `AWS_...` variables:
`AWS_KEY, AWS_SECRET, AWS_BUCKET`

File: scripts/env-check.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "AWS_KEY",
+  "AWS_SECRET",
+  "AWS_BUCKET",
+];
+
+const docUrl = 'https://github.com/jexia/jexia-sdk-js/blob/master/e2e/README.md';
```
Comment (KutsenkoA) [inline scripts/env-check.js]: still single quotes :-D

Comment (marlosin) [reply scripts/env-check.js]: :weary:

**Pull request merged**